Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu/mysql:8.0-20.04_beta
+
+ENV CONFIG_VERSION="0.8.22-1"
+
+RUN set -eux && \
+	apt-get update && \
+	apt-get install -y wget lsb-release gnupg
+
+RUN set -eux && \
+	wget -q https://dev.mysql.com/get/mysql-apt-config_${CONFIG_VERSION}_all.deb \
+        -O /tmp/mysql-apt-config_${CONFIG_VERSION}_all.deb && \
+    DEBIAN_FRONTEND=noninteractive dpkg -i /tmp/mysql-apt-config_${CONFIG_VERSION}_all.deb && \
+    DEBIAN_FRONTEND=noninteractive apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y mysql-shell && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
This isn't going to clear up how you think - these lists were originally created in the last layer (line 6). You'll need to combine these two `RUN` directives into one for this cleanup to work properly

Reply:
combined in 69ec1f179f718412815e44d05e7e846f96774445